mods/cnc/maps/gdi10a/gdi10a.lua

```diff
@@ -0,0 +1,413 @@
+--[[
+   Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+Difficulty = Map.LobbyOption("difficulty")
```

Comment:
```suggestion
```
This should already be covered by the global `campaign.lua`.

Reply:
Yes, when I started work on this map, it was looking in the bits folder.  Removed my global `Difficulty`.